apps/condo/domains/onboarding/components/WelcomePopup.tsx

```diff
@@ -0,0 +1,192 @@
+    const intl = useIntl()
+    const TitleMessage = intl.formatMessage({ id: 'WelcomePopup.title' })
+
+    return (
+        <Row justify={'space-between'}>
+            <Col>
+                <Typography.Title level={3}>{TitleMessage}</Typography.Title>
+            </Col>
+            <Col>
+                <CrossIcon onClick={handleModalClose} css={crossIconStyles}/>
+            </Col>
+        </Row>
+    )
+}
+
+const MODAL_ROW_STYLES = { padding: '20px 40px' }
+
+const ModalFooter = ({ step, setStep, handleModalClose }) => {
+    const intl = useIntl()
+    const OkMessage = intl.formatMessage({ id: 'OK' })
+
+    const handleNextStepButtonClick = useCallback(() => setStep(currentStep => currentStep + 1), [setStep])
+    const handlePrevStepButtonClick = useCallback(() => setStep(currentStep => currentStep - 1), [setStep])
+
+    return (
+        <Row style={MODAL_ROW_STYLES} justify={'space-between'}>
+            <Col>
+                <Row gutter={[20, 0]}>
+                    <Col>
+                        <Button onClick={handlePrevStepButtonClick} disabled={step === 0} type={'text'}>
+                            ←
+                        </Button>
+                    </Col>
+                    <Col>
+                        <Button onClick={handleNextStepButtonClick} disabled={step === 2} type={'text'}>
+                            →
+                        </Button>
+                    </Col>
+                </Row>
+            </Col>
+            {
+                step === 2 && (
+                    <Button type={'sberDefaultGradient'} onClick={handleModalClose}>
+                        {OkMessage}
+                    </Button>
+                )
+            }
+        </Row>
+    )
+}
+
+type WelcomePopupImage = {
+    src: string
+    style: CSSProperties
+}
+
+type WelcomePopupStep = {
+    imageBackgroundColor: string
+    images: WelcomePopupImage[]
+    title: string
+    text: string
+}
+
+const IMAGE_COL_STYLES = { display: 'flex', alignItems: 'end' }
+const BODY_TEXT_STYLES = { fontSize: fontSizes.content, lineHeight: '24px', letterSpacing: '-0.01em' }
+
+export function WelcomePopup () {
+    const intl = useIntl()
+    const Step1Title = intl.formatMessage({ id: 'WelcomePopup.step1.title' })
+    const Step1Text = intl.formatMessage({ id: 'WelcomePopup.step1.text' })
+    const Step2Title = intl.formatMessage({ id: 'WelcomePopup.step2.title' })
+    const Step2Text = intl.formatMessage({ id: 'WelcomePopup.step2.text' })
+    const Step3Title = intl.formatMessage({ id: 'WelcomePopup.step3.title' })
+    const Step3Text = intl.formatMessage({ id: 'WelcomePopup.step3.text' })
+
+    const [step, setStep] = useState(0)
+    const [visible, setVisible] = useState(true)
+
+    const handleModalClose = useCallback(() => setVisible(false), [setVisible])
+
+    const stepToContent: WelcomePopupStep[] = useMemo(() => ([
+        {
+            imageBackgroundColor: '#E4EFFF',
```

Comment:
shouldn't we use predefined colors from library?

Reply:
moved colors to constants and import them here